static/css/mozsearch.css

```diff
@@ -1383,53 +1411,74 @@ input::placeholder {
  * Our diagram pointer background colors come from the ColorBrewer2 diverging
  * PiYG scheme with 11 steps: https://colorbrewer2.org/#type=diverging&scheme=PiYG&n=11
  */
+
+@supports (color-scheme: dark) {
+@media (prefers-color-scheme: dark) {
+
+svg [fill="white"] {
+  fill: var(--page-background);
+}
+svg [stroke="black"] {
+  stroke: var(--page-foreground);
+}
+svg text {
+  fill: var(--page-foreground);
+}
+
+polygon[fill="#eeeeee"] {
+  fill: var(--panel-header-background);
+}
+
+}
+}
+
 g.ptr-strong > polygon {
-  fill: #7fbc41;
+  fill: var(--diagram-ptr-strong);
 }
 g.ptr-unique > polygon {
-  fill: #b8e186;
+  fill: var(--diagram-ptr-unique);
 }
 g.ptr-weak > polygon {
-  fill: #fde0ef;
+  fill: var(--diagram-ptr-weak);
 }
 g.ptr-raw > polygon {
-  fill: #f1b6da;
+  fill: var(--diagram-ptr-raw);
 }
 g.ptr-ref > polygon, g.ptr-contains > polygon {
-  fill: #e6f5d0;
+  fill: var(--diagram-ptr-ref);
 }
 
 g.hovered-cur-node > polygon {
-  fill: #fbf;
+  fill: var(--diagram-hovered-cur-node);
 }
 g.hovered-out-node > polygon {
-  fill: #bbf;
+  fill: var(--diagram-hovered-out-node);
 }
 g.hovered-in-node > polygon {
-  fill: #bfb;
+  fill: var(--diagram-hovered-in-node);
 }
 
 g.hovered-cur-edge > path:not(.clicktarget),
 g.hovered-cur-edge > polygon {
-  stroke: #eae;
+  stroke: var(--diagram-hovered-cur-edge);
   stroke-opacity: 1;
   stroke-width: 2px;
 }
 
 g.edge > path.clicktarget {
-  stroke:rgba(0,0,0,0);
+  stroke: var(--diagram-edge-clicktarget);
   stroke-width: 10px;
 }
 
 g.edge.hovered-out-edge > path,
 g.edge.hovered-out-edge > polygon {
-  stroke: blue;
+  stroke: var(diagram-hovered-out-edge--);
```

Comment:
@arai-a When reviewing the ignore nodes patch with local testing I just discovered the effects of this typo and therefore this typo.  Not sure if you want to add a fix in a stack or just do a separate immediately landed PR or what not.

Reply:
Thanks!
I'll fix it in the ignore nodes patch stack